bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -0,0 +1,144 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
```

Comment:
Only reason I haven't done this yet is I'm not sure all the meta stuff *should* be shared. I want to run the new pages by both garethc and habber to see if the scenes should have different titles, descriptions, etc.

Based on sprint timing, envisioning this as a task for next sprint - either customize meta info per scene/page, or create shared base template.